lib/ZInit.c

```diff
@@ -196,6 +199,20 @@ ZInitialize(void)
 	    close(s);
 	}
     }
+#if defined(__APPLE__) && defined(__MACH__)
+    if (__My_addr.s_addr == INADDR_NONE) {
+      nwi_state_t state;
+      state = nwi_state_copy();
```

Comment:
Why not merge these two lines?

Reply:
No good reason other than trying to copy local style.